### ISSUE TITLE
Edit score color variables

### ISSUE THREAD

**Issue by hedgefield** [Yoast/yoast-components#655]
In the new Gutenberg integrations, the colors I used for the score icons are not the same as what is recorded in [our colors scss](https://github.com/Yoast/yoast-components/blob/develop/style-guide/_colors.scss).

![colors bullets old](https://user-images.githubusercontent.com/6073772/42510900-a1e3d216-8450-11e8-8384-c08670dda9a6.png)
![colors bullets new](https://user-images.githubusercontent.com/6073772/42510901-a1fb2d8a-8450-11e8-89be-f6a0ad284e5d.png)

I'm apprehensive to changing the colors for all the bullets at once, but the new colors do have better colorblind contrast with each other, plus the new smileys. So I want to ask @afercia to give another review, and if he agrees, to change the following variables:

Create new `palette_yellow_medium` with hex color `#f5c819`.
Change `$color_ok` from `palette_orange_light` to `palette_yellow_medium`.
Change `$color_good` from `palette_green_light` to `palette_green_medium_light`.
Change `$color_score_icon` from `palette_grey_medium_dark` to `palette_grey_medium`.

**Comment by afercia** [comment]
Well the point is if there's no other textual indication and we want to rely only on the score icons, then we have to consider the "smileys" as text against a background. The minimum luminosity contrast is the usual one: 4.5:1
The gray icon has no "text" so it should only meet the requirement for UI controls agains their background, which is 3:1

Though the new green increases a bit the luminosity contrast, all the new colors are still under the requirement.

Contrast ratio against white text, it should be at least 4.5:1:
- `#f5c819` (yellow): 1.6:1
- `#64a60a` (green): better but still 3:1
- `#dc3232` (red): unchanged 4.62:1

Against the white background of the UI, it should be at least 3:1:
- `#cccccc` (grey): the contrast decreases from the one of the previous gray 3.54:1 to 1.61:1

Unsure what to suggest. The yellow / orange case is the more difficult one. What about trying very light backgrounds and black smileys? Or maybe, use just a colored red/yellow/green rounded border , a white background, and black text?

**Comment by afercia** [comment]
> Would you agree?
I agree it's an improvement 😉

**Comment by hedgefield** [comment]
Great, [I've made SVGs which have the color baked in](https://github.com/Yoast/yoast-components/files/2188530/color_smileys.zip), we can start using those and go from there.
